app/src/main/java/io/github/polysmee/invites/fragments/AppointmentsInvitesFragment.java

```diff
@@ -0,0 +1,257 @@
+package io.github.polysmee.invites.fragments;
+
+import android.content.Intent;
+import android.os.Bundle;
+
+import androidx.constraintlayout.widget.ConstraintLayout;
+import androidx.fragment.app.Fragment;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import java.text.MessageFormat;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Set;
+
+import io.github.polysmee.R;
+import io.github.polysmee.agora.Command;
+import io.github.polysmee.appointments.AppointmentActivity;
+import io.github.polysmee.calendar.CalendarAppointmentInfo;
+import io.github.polysmee.calendar.fragments.CalendarActivityFragmentsHelpers;
+import io.github.polysmee.calendar.googlecalendarsync.CalendarUtilities;
+import io.github.polysmee.database.Appointment;
+import io.github.polysmee.database.DatabaseAppointment;
+import io.github.polysmee.database.User;
+import io.github.polysmee.database.databaselisteners.StringSetValueListener;
+import io.github.polysmee.login.MainUser;
+
+public class AppointmentsInvitesFragment extends Fragment {
+
+    private StringSetValueListener userInvitesListener;
+
+    private LinearLayout scrollLayout;
+    private LayoutInflater inflater;
+    private ViewGroup rootView;
+
+    private User user;
```

Comment:
If it going to only store the main user it might be a better idea for clarity to declare the variable as "private User mainUser"